### ISSUE TITLE
[Bug] operator wrongly uses `storageSpec.name` in the value.yaml set the wrong filed in the manifest

### ISSUE THREAD

**Issue by dengliu** [StarRocks/starrocks-kubernetes-operator#52]
When setting `storageSpec.name` in the value.yaml file, it actually initialized it as the volume name instead of storageclassName in the eventual manifest.

values.yaml
```
    # fe storageSpec for persistent meta data.
    storageSpec:
      # the storageClass name for fe meta persistent volume. if name is empty used emptyDir.
      name: "gp2" <====================
```

generate statefulset manifest
```
  volumeClaimTemplates:
  - apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      creationTimestamp: null
      name: gp2 <===================== mistakenly set it as the volume name
    spec:
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 100Gi
      volumeMode: Filesystem
      storageClassName: gp2 <=================== should set spec.storageClassName

```

**Comment by dengliu** [comment]
@kevincai @shileifu 
Can you please prioritize this fix? Thanks a lot!

**Comment by shileifu** [comment]
yes, you are right. the code use name as storageclass name and PVC objectname. the filed storageClassName in StarRocksCluster CRD is not used. i will fix it.

**Comment by shileifu** [comment]
have fixed in 1.3.4

**Comment by dengliu** [comment]
Thanks. Have verified it is fixed in 1.3.5.
